### DISCUSSION TITLE
Multiples ops/callsigns

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#1931] [Q&A]
Hello!

What is the best way to handle multiple ops/callsigns? I see the feature for "Station Logbooks", but are they completely seperate things? I am looking for the option to log club QSOs using a different callsign.

Thanks
Scott, W1AL

**Comment by magicbug** [comment]
At the moment it probably won't do what you are after for Club QSOs, however, its coming, a member will need their own account and they will set up a Station Location as the club callsign this can then be shared via the Station Logbooks feature to link it all together.

Most of the codes there its interface that needs to be built.

**Comment by soakland5** [comment]
Thanks!

I have another question: I am trying to sync / import my LOTW
confirmations, and it breaks each time, even when I try and make the date
range for QSOs smaller.

Ideas?
